docs/contributing/overview.md

```diff
@@ -262,26 +262,6 @@ The `--autogenerate` flag will automatically generate a migration file based on
     Be sure to check the file to make sure it only includes the changes you want to make. Additionally, you may need to
     remove extra statements that were included and not related to your change.
 
-When adding a migration for SQLite, it's important to include the following `PRAGMA` statements for both upgrade and downgrade:
-
-```python
-def upgrade():
-    op.execute("PRAGMA foreign_keys=OFF") # <-- add this line
-    
-    # migration code here
-    
-    op.execute("PRAGMA foreign_keys=ON") # <-- add this line
-
```

Comment:
probably will need to remove from the migration template as well:

https://github.com/PrefectHQ/prefect/blob/main/src/prefect/server/database/migrations/script.py.mako